selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
I still disagree as this is important test unveiling a fork-bomb. When Avocado works properly it takes 0.4s. The timeout 5s was introduced to avoid OOM killer when Avocado misbehaves and can be increased or even removed completely as properly functioning Avocado should simply FAIL the simple TEST.